### PR TITLE
Task 2

### PR DESCRIPTION
**Pull request by parabolabam**
cloud fron link https://db5z5p02d6kgh.cloudfront.net

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -74,7 +74,7 @@ resources:
       Properties:
         DistributionConfig:
           Origins:
-            - DomainName: !GetAtt WebAppS3Bucket.RegionalDomainName
+            - DomainName: https://aws-react-bucket-vsorokin.s3.amazonaws.com
```
Comment (faichuk-volodymyr) [inline serverless.yml]: Why it's replaced?

File: .serverless_plugins/serverless-single-page-app-plugin.js [hunk 2]
```diff
@@ -72,29 +72,39 @@ class ServerlessPlugin {
...
+    // throw error;
+
+    // code above throws an error as CloudFront returns [
+    //   {
+    //     OutputKey: 'ServerlessDeploymentBucketName',
+    //     OutputValue: 'my-store-app-dev-serverlessdeploymentbucket-1kqb34k5yxa53'
+    //   }
+    // ]
+    // taken from this https://us-east-1.console.aws.amazon.com/cloudfront/v3/home?region=us-east-1#/distributions
+    return "db5z5p02d6kgh.cloudfront.net";
```
Comment (faichuk-volodymyr) [inline .serverless_plugins/serverless-single-page-app-plugin.js]: Should it be hardcoded in this way?

File: README.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+CloundFront link https://db5z5p02d6kgh.cloudfront.net/
+S3 link https://aws-react-bucket-vsorokin.s3.amazonaws.com/
```
Comment (faichuk-volodymyr) [inline README.md]: It shouldn't return XML document

Comment (faichuk-volodymyr) [reply README.md]: https://aws-react-bucket-vsorokin.s3.amazonaws.com/index.html it stell returns index.html

File: README.md [hunk 4]
```diff
@@ -1,3 +1,6 @@
+CloundFront link https://db5z5p02d6kgh.cloudfront.net/
```
Comment (faichuk-volodymyr) [inline README.md]: This link is a result of manual deploying or automated deploying?

File: README.md (hunk 3 above)
Comment (ex4twgh) [inline README.md]: It does not show SPA app, only message NoSuchBucket

File: serverless.yml (hunk 1 above)
Comment (parabolabam) [reply serverless.yml]: Returned. Was into ambigious situation

File: .serverless_plugins/serverless-single-page-app-plugin.js (hunk 2 above)
Comment (parabolabam) [reply .serverless_plugins/serverless-single-page-app-plugin.js]: no, you're right, but it was throwing due to misconfiguration

File: README.md (hunk 3 above)
Comment (parabolabam) [reply README.md]: links updated,

File: README.md (hunk 4 above)
Comment (parabolabam) [reply README.md]: It's all auto deployed,

File: README.md (hunk 3 above)
Comment (parabolabam) [reply README.md]: you may recheck it one more tim please, becasue I updated links

**Pull request merged**